docs/conf.py

```diff
@@ -76,13 +67,42 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
     'sphinx.ext.mathjax',
-    'recommonmark',
-    'sphinx_markdown_tables',
     'sphinx_togglebutton',
+    'myst_parser',
     'notfound.extension',
 ]
 
-master_doc = 'index'
+myst_enable_extensions = [
+    "amsmath",
+    "attrs_inline",
+    "colon_fence",
+    "deflist",
+    "dollarmath",
+    "fieldlist",
+    "html_admonition",
+    "html_image",
+    "replacements",
+    "smartquotes",
+    "strikethrough",
+    "substitution",
+    "tasklist",
+]
+
+myst_substitutions = {
+  "release": release,
+  "kyuubi_hive_jdbc_shaded_maven_ref":
+"""
+```xml
+<dependency>
+    <groupId>org.apache.kyuubi</groupId>
+    <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
+    <version>%s</version>
```

Comment:
substitution does not support code block, so we need to define to whole block instead